assets/js/store.js

```diff
@@ -62,12 +62,26 @@ Store.prototype.has = function (key/*: string*/)/*: bool*/ {
 
 Store.prototype.able = function ()/*: bool*/ {
     try {
+        throwLocalStorageNotAccessibleError();
```

Comment:
> Doesn't this always throw the exception? 

Yep. I tested it incorrectly. My bad!

>  I meant catching the SecurityException that you showed Firefox throws, not creating your own exception.

I will look for the error codes that exist for such exceptions! 
Thanks :D